src/components/backgrounds/BackgroundPreviewSettings.tsx

```diff
@@ -66,7 +67,12 @@ const BackgroundPreviewSettings = ({
   );
   const [isOpen, setIsOpen] = useState<boolean>(false);
   const [buttonFocus, setButtonFocus] = useState<boolean>(false);
-  const value = useAppSelector((state) => state.editor.previewAsSceneId);
+  const value = useAppSelector((state) => {
+    if (state.project.present.settings.previewAsMono) return "0";
+    const sceneId = state.editor.previewAsSceneId;
+    if (sceneId === "") return "1";
+    return sceneId;
+  });
```

Comment:
I added a mono mode, similar to the one found on the world page, to the "preview as" drop down. Previously, only one non-scene option was available in this drop down, "Default Colors", which was used when `previewAsSceneId` was "". To facilitate more options, I'm just using the stringified index of the non-scene values. To ensure a consistent experience when switching between pages, I set the default state to "0" when `previewAsMono` and "1" when `previewAsSceneId` is unset.